tests/analysis.spec.ts

```diff
@@ -301,30 +301,50 @@ describe('Functional test of analysis', () => {
         analysisContext: {
           flow: 'test',
           initiator: 'CLI',
-          orgDisplayName: 'org',
-          orgPublicId: 'id',
-          projectName: 'proj',
-          projectPublicId: 'id',
+          org: {
+            name: 'org',
+            displayName: 'organization',
+            publicId: 'id',
+            flags: {},
+          },
+          project: {
+            name: 'proj',
+            publicId: 'id',
+            type: 'code',
+          },
         },
       };
 
       const makeRequestSpy = jest.spyOn(needle, 'makeRequest');
 
-      await analyzeFolders({
```

Comment:
q: Why change this from `analyzeFolders` to `analyzeBundle`?

Reply:
otherwise this test will not be able to call "/analysis", as it will break at bundle creation step.
Since i did not want to change the purpose of the test, this was the way.
Btw, let me add a new test for /bundle calls only